torch/testing/_internal/common_methods_invocations.py

```diff
@@ -345,6 +345,94 @@ def __init__(self,
         #   outside a function's domain.
         self._domain_eps = 1e-5
 
+
+def sample_inputs_binary(op_info, device, dtype, requires_grad):
+    low, high = op_info.domain
+    low = low if low is None else low + op_info._domain_eps
+    high = high if high is None else high - op_info._domain_eps
+
+    return (
+        # empty tensors
+        SampleInput((make_tensor((), device, dtype,
+                                 low=low, high=high,
+                                 requires_grad=requires_grad),
+                     make_tensor((), device, dtype,
+                                 low=low, high=high,
+                                 requires_grad=requires_grad))),
+        # 1d - 1d
+        SampleInput((make_tensor((L), device, dtype,
```

Comment:
Let's swap L for M to lower test time slightly (we may revise our input sizes again after @soulitzer improves grad{grad}check performance)

Reply:
ok